pkg/tenant/members_test.go

```diff
@@ -493,11 +503,6 @@ func (suite *tenantTestSuite) TestMemberUpdate() {
 	data, err := member.MarshalValue()
 	require.NoError(err, "could not marshal the member")
 
-	// Unmarshal the data with msgpack
```

Comment:
Should this be removed from the other test files as well?

Reply:
Yes thanks I'll include it in the story.